SlowCheetah.JDT.Tests/TransformTest.cs

```diff
@@ -0,0 +1,179 @@
+        private static readonly string RemoveTestDirectory = TestInputDirectory + "Remove\\";
+
+        // Directory for Rename transformation test
+        private static readonly string RenameTestDirectory = TestInputDirectory + "Rename\\";
+
+        // Directory for Replace transformation test
+        private static readonly string ReplaceTestDirectory = TestInputDirectory + "Replace\\";
+
+        // Directory for Merge transformation test
+        private static readonly string MergeTestDirectory = TestInputDirectory + "Merge\\";
+
+        /// <summary>
+        /// Gets inputs for the Default transformation
+        /// </summary>
+        public static IEnumerable<object[]> GetDefaultInputs
+        {
+            get
+            {
+                // Gets inputs from Default transformation test directory
+                return GetInputs(DefaultTestDirectory);
+            }
+        }
+
+        /// <summary>
+        /// Gets inputs for the Remove transformation
+        /// </summary>
+        public static IEnumerable<object[]> GetRemoveInputs
+        {
+            get
+            {
+                return GetInputs(RemoveTestDirectory);
+            }
+        }
+
+        /// <summary>
+        /// Gets inputs for the Rename transformation
+        /// </summary>
+        public static IEnumerable<object[]> GetRenameInputs
+        {
+            get
+            {
+                return GetInputs(RenameTestDirectory);
+            }
+        }
+
+        /// <summary>
+        /// Gets inputs for the Replace transformation
+        /// </summary>
+        public static IEnumerable<object[]> GetReplaceInputs
+        {
+            get
+            {
+                return GetInputs(ReplaceTestDirectory);
+            }
+        }
+
+        /// <summary>
+        /// Gets inputs for the Merge transformation
+        /// </summary>
+        public static IEnumerable<object[]> GetMergeInputs
+        {
+            get
+            {
+                return GetInputs(MergeTestDirectory);
+            }
+        }
+
+        /// <summary>
+        /// Tests the Default transformation
+        /// </summary>
+        /// <param name="testFileName">Name of the test being performed.
+        /// Corresponds to a group of files in the input folder</param>
+        [Theory]
+        [MemberData(nameof(GetDefaultInputs))]
+        public void Default(string testFileName)
+        {
+            BaseTransformTest(DefaultTestDirectory, testFileName);
+        }
+
+        /// <summary>
+        /// Tests the Remove transformation
+        /// </summary>
+        /// <param name="testFileName">Name of the test being performed.
+        /// Corresponds to a group of files in the input folder</param>
+        [Theory]
+        [MemberData(nameof(GetRemoveInputs))]
+        public void Remove(string testFileName)
+        {
+            BaseTransformTest(RemoveTestDirectory, testFileName);
```

Comment:
I haven't deeply studied your unit tests, but it at least appears that your tests are super small and concise. That's *great* as that tends to lead to more tests being written now and in the future when folks add functionality or fix bugs and ought to add regression tests.